ref/config/serverConfiguration.adoc

```diff
@@ -380,20 +307,13 @@ conflicting config or to replace it (`REPLACE`).
 </server>
 ----
 
-You can set the `location` attribute to a relative or absolute file path or to
-an HTTP URL.
+You can set the `location` attribute to a relative or absolute file path, or to an HTTP URL.
+
 
-=== Config references
-Most configuration in {projectName} is self-contained but it is often useful to
-be able to share config. A common example of this would be the JDBC driver
-config being shared by multiple data sources, or sharing the classloader for
-JDBC driver classes so the classes are visible both to the DataSource and to an
-application. Any factory config element defined as a direct child of the
-`server` element can be referred to.
+== Configuration references
+Most configuration in Open Liberty is self-contained but it can be useful to share config. For example, the JDBC driver config might be shared by multiple data sources. You can refer to any factory config element that is defined as a direct child of the `server` element.
```

Comment:
Open Liberty should be {projectName}

The rephrasing reduces the utility of the documentation. Before it says "here is something many people need/want to do" and it goes to "something that some people might need, but who knows if it is common". I prefer the original wording because it enhances the utility of the information being conveyed.

Reply:
Updated